go.mod

```diff
@@ -5,4 +5,10 @@ require (
 	github.com/stretchr/testify v1.8.0
 )
 
-go 1.13
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+go 1.18
```

Comment:
github.com/pkg/errors can be updated to 0.9.1 and also move 'go 1.18' line to the top of the page